src/react-apps/applications/service-development/src/features/deploy/containers/deployToTestContainer.tsx

```diff
@@ -0,0 +1,179 @@
+import { Grid, Hidden, Paper, Typography } from '@material-ui/core';
+import { createMuiTheme, createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import altinnTheme from '../../../../../shared/src/theme/altinnStudioTheme';
+import { getLanguageFromKey } from '../../../../../shared/src/utils/language';
+import postMessages from '../../../../../shared/src/utils/postMessages';
+import DeployPaper from '../components/deployPaper';
+import DeployActionDispacher from '../deployDispatcher';
+
+const theme = createMuiTheme(altinnTheme);
+
+const env = 'at21';
+
+const styles = () => createStyles({
+  mainLayout: {
+    paddingTop: 20,
+    [theme.breakpoints.up('md')]: {
+      height: `calc(100vh - 110px)`,
+      overflowY: 'auto',
+      paddingLeft: theme.sharedStyles.mainPaddingLeft,
+    },
+    [theme.breakpoints.down('sm')]: {
+      height: `calc(100vh - 55px)`,
+      overflowY: 'auto',
+    },
+  },
+  ingress: {
+    paddingTop: 10,
+  },
+  deployPlaceholderStyle: {
+    marginTop: 24,
+    [theme.breakpoints.up('md')]: {
+      paddingRight: 60,
+    },
+  },
+  aboutServicePlaceholderStyle: {
+    [theme.breakpoints.up('md')]: {
+      borderLeft: '1px solid ' + theme.altinnPalette.primary.greyMedium,
+      paddingLeft: 12,
+    },
+    marginTop: 24,
+  },
+  mainGridStyle: {
+    maxWidth: 1340,
+  },
+});
+
+export interface IDeployToTestContainerProps extends WithStyles<typeof styles> {
+  deploymentList: any;
+  language: any;
+  masterRepoStatus: any;
+  name?: any;
+  repoStatus: any;
+}
+
+export interface IDeployToTestContainerState {
+  deploySuccess: boolean;
+}
+
+export class DeployToTestContainer extends
+  React.Component<IDeployToTestContainerProps, IDeployToTestContainerState> {
+
+  constructor(_props: IDeployToTestContainerProps, _state: IDeployToTestContainerState) {
+    super(_props, _state);
+    this.state = {
+      deploySuccess: null,
+    };
+  }
+
+  public componentDidMount() {
+    const altinnWindow: any = window;
+    const { org, service } = altinnWindow;
+    DeployActionDispacher.fetchDeployments(env, org, service);
+    DeployActionDispacher.fetchMasterRepoStatus('TODO', org, service);
+    window.postMessage(postMessages.forceRepoStatusCheck, window.location.href);
+  }
+
+  public returnInSyncStatus = (repoStatus: any): any => {
+    if (repoStatus.contentStatus) {
+      if (repoStatus.contentStatus.length > 0) {
+        return 'ahead';
```

Comment:
can we use enums or constant for these? I see they are also referenced in deployPaper

Reply:
How do you propose we do that?

Reply:
Typescript has enums, right? Isnt there a way to use that? So you can write something like:
if (repoStatus.contentStatus.length > 0) {
        return Repostatus.ahead;

Reply:
Added enum.